flickr api key mandatory

In the local.py the flickr_api_key and flickr_api_secret are defined based on the config.json values, but they are mandatory, even if the photo repository is local and there are no flickr key and api codes. Results in error when doing make migrate and make index

```python
ubuntu@voseq2test4:~/VoSeq$ make migrations
python voseq/manage.py makemigrations --settings=voseq.settings.local
Traceback (most recent call last):
  File "/home/ubuntu/VoSeq/voseq/voseq/settings/local.py", line 25, in get_secret
    return secrets[setting]
KeyError: 'FLICKR_API_KEY'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "voseq/manage.py", line 10, in <module>
    execute_from_command_line(sys.argv)
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/__init__.py", line 338, in execute_from_command_line
    utility.execute()
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/__init__.py", line 330, in execute
    self.fetch_command(subcommand).run_from_argv(self.argv)
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/__init__.py", line 190, in fetch_command
    klass = load_command_class(app_name, subcommand)
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/__init__.py", line 40, in load_command_class
    module = import_module('%s.management.commands.%s' % (app_name, name))
  File "/usr/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 2231, in _gcd_import
  File "<frozen importlib._bootstrap>", line 2214, in _find_and_load
  File "<frozen importlib._bootstrap>", line 2203, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 1200, in _load_unlocked
  File "<frozen importlib._bootstrap>", line 1129, in _exec
  File "<frozen importlib._bootstrap>", line 1448, in exec_module
  File "<frozen importlib._bootstrap>", line 321, in _call_with_frames_removed
  File "/usr/local/lib/python3.4/dist-packages/django/core/management/commands/makemigrations.py", line 8, in <module>
    from django.db.migrations.autodetector import MigrationAutodetector
  File "/usr/local/lib/python3.4/dist-packages/django/db/migrations/autodetector.py", line 13, in <module>
    from django.db.migrations.questioner import MigrationQuestioner
  File "/usr/local/lib/python3.4/dist-packages/django/db/migrations/questioner.py", line 12, in <module>
    from .loader import MIGRATIONS_MODULE_NAME
  File "/usr/local/lib/python3.4/dist-packages/django/db/migrations/loader.py", line 10, in <module>
    from django.db.migrations.recorder import MigrationRecorder
  File "/usr/local/lib/python3.4/dist-packages/django/db/migrations/recorder.py", line 9, in <module>
    class MigrationRecorder(object):
  File "/usr/local/lib/python3.4/dist-packages/django/db/migrations/recorder.py", line 23, in MigrationRecorder
    class Migration(models.Model):
  File "/usr/local/lib/python3.4/dist-packages/django/db/migrations/recorder.py", line 24, in Migration
    app = models.CharField(max_length=255)
  File "/usr/local/lib/python3.4/dist-packages/django/db/models/fields/__init__.py", line 1081, in __init__
    super(CharField, self).__init__(*args, **kwargs)
  File "/usr/local/lib/python3.4/dist-packages/django/db/models/fields/__init__.py", line 161, in __init__
    self.db_tablespace = db_tablespace or settings.DEFAULT_INDEX_TABLESPACE
  File "/usr/local/lib/python3.4/dist-packages/django/conf/__init__.py", line 48, in __getattr__
    self._setup(name)
  File "/usr/local/lib/python3.4/dist-packages/django/conf/__init__.py", line 44, in _setup
    self._wrapped = Settings(settings_module)
  File "/usr/local/lib/python3.4/dist-packages/django/conf/__init__.py", line 92, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python3.4/importlib/__init__.py", line 109, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "/home/ubuntu/VoSeq/voseq/voseq/settings/local.py", line 35, in <module>
    FLICKR_API_KEY = get_secret("FLICKR_API_KEY")
  File "/home/ubuntu/VoSeq/voseq/voseq/settings/local.py", line 28, in get_secret
    raise ImproperlyConfigured(error_msg)
django.core.exceptions.ImproperlyConfigured: Set the FLICKR_API_KEY environment variable
make: *** [migrations] Error 1
ubuntu@voseq2test4:~/VoSeq$
```

--- a/voseq/voseq/settings/local.py
+++ b/voseq/voseq/settings/local.py
@@ -32,8 +32,9 @@ SECRET_KEY = get_secret("SECRET_KEY")
 GOOGLE_MAPS_API_KEY = get_secret("GOOGLE_MAPS_API_KEY")
 
 PHOTOS_REPOSITORY = get_secret("PHOTOS_REPOSITORY")
-FLICKR_API_KEY = get_secret("FLICKR_API_KEY")
-FLICKR_API_SECRET = get_secret("FLICKR_API_SECRET")
+if PHOTOS_REPOSITORY == 'flickr':
+    FLICKR_API_KEY = get_secret("FLICKR_API_KEY")
+    FLICKR_API_SECRET = get_secret("FLICKR_API_SECRET")
 
 DATABASES = {
     'default': {

